code/src/components/Album.js

```diff
@@ -0,0 +1,28 @@
+import React from 'react';
+import Icons from './Icons';
```

Comment:
The` Icons` component is imported but not used inside the Album component. It might be better to remove the import statement.